env/build_hera_intel.env

```diff
@@ -2,42 +2,54 @@
 
 module purge
 
+# UFS_UTILS
+module load hpss
+module load cmake/3.16.1
```

Comment:
Should # UFS_UTILS be listed above these two modules?  HPSS is only used for the exregional_get_extrn_mdl_files.sh script and cmake is for everything.

Reply:
@JeffBeck-NOAA, The comment means the order of UFS_UTILS -> ufs-weather-model -> EMC_post (order in Externals.cfg). It doesn't mean a specific component. This is the reason why I put '+' before EMC_post or ufs-weather-model in comments (for example, # + EMC_post). I used it as an indicator of 'in addition'. If you think these comments make users confused, I'll remove all the comments and rearrange the modules in the alphabetical order.

Reply:
Thanks, @chan-hoo.  I'm honestly still confused, but it could just be me.  I was thinking they were block headers that represent which modules apply to which App component. How do they represent the Externals.cfg file?

Reply:
@JeffBeck-NOAA, I've removed all the comments about components and rearranged modules only in alphabetical order. I'll test building on each machine again.

Reply:
@JeffBeck-NOAA, completed!

Reply:
Thank you @chan-hoo!  This looks good to me.  Going to approve, but would like to have @mkavulich's review as well.